Fabric.Identity.API/Infrastructure/PolicyProvider.cs

```diff
@@ -11,7 +13,17 @@ public class PolicyProvider
         public readonly CircuitBreakerPolicy LdapErrorPolicy = Policy.Handle<LdapException>()
             .CircuitBreaker(5, TimeSpan.FromMinutes(5));
 
+        public readonly RetryPolicy IdPSearchServiceRetryPolicy = Policy.Handle<HttpRequestException>()
+            .RetryAsync(3);
+
         public readonly CircuitBreakerPolicy IdPSearchServiceErrorPolicy = Policy.Handle<HttpRequestException>()
-            .CircuitBreaker(5, TimeSpan.FromMinutes(5));
+            .CircuitBreakerAsync(3, TimeSpan.FromMinutes(1));
```

Comment:
Is there a reason we want to close the circuit after one minute instead of 5?

Reply:
I figured 5 minutes was a rather long time to leave the circuit open, especially based on how this bug surfaces, but I can switch it back.